cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -65,6 +72,11 @@ type node struct {
 	tenantCRSs map[*node]empty
 }
 
+type crdInfo struct {
+	typeMeta  metav1.TypeMeta
+	moveLabel bool
```

Comment:
```suggestion
	forceMove bool
```
So we are not tight to the mechanisms used to signal that all the objects of this type should be moved (in API machinery there are discussion about if it is legit to apply labels to CRDs 🤔 )